components/molecules/DSHeader.js

```diff
@@ -4,41 +4,37 @@ import fr from '../../locales/fr'
 
 export default function DSHeader(props) {
   const t = props.locale === 'en' ? en : fr
-  const menuItems = [
-    {
-      link: '/dashboard',
-      text: 'Services',
-    },
-    {
-      link: '/dashboard',
-      text: 'Life Events',
-    },
-    {
-      link: '/dashboard',
-      text: 'Contact us',
-    },
-  ]
   return (
     <Header
-      menuItems={menuItems}
-      breadCrumbItems={{
-        id: 'breadcrumbID',
-        items: [
-          {
-            link: t.url_canada_ca,
-            text: t.canada_ca,
-          },
-          {
-            link: t.url_serviceCanada,
-            text: t.serviceCanada,
-          },
-          {
-            link: t.url_myBenefitsAndServices,
-            text: t.myBenefitsAndServices,
-          },
-        ],
+      id="header"
+      lang={t}
```

Comment:
We shouldn't pass our language file, just 'en' or 'fr' depending on the language.
```suggestion
      lang={props.locale}
```

Reply:
Done, thank you!